test/processes.test.js

```diff
@@ -20,6 +31,71 @@ describe('Launch all processes', () => {
     const procNames = Object.keys(procs).sort();
     assert.deepEqual('Spawns app, phantom, and proxy',
       ['application', 'phantomjs', 'proxy'], procNames);
-    each(procs, ({ rawProcess }) => rawProcess.kill());
+    killProcs(procs);
+  });
+});
+
+describe('Launch all processes Chrome', () => {
+  const chromeOptions = [
+    '--disable-application-cache',
+    '--media-cache-size=1',
+    '--disk-cache-size=1',
+    '--disk-cache-dir=/dev/null',
+    '--disable-cache',
+    '--disable-desktop-notifications',
+    '--headless'];
+
+  let config;
+  beforeEach(() => {
+    config = new Config({
+      root: HELLO_WORLD,
+      launch: true,
+      browser: 'chrome',
+    });
+  });
+
+  afterEach(() => {
+    config = null;
+  });
+
+  it('launches all the processes', async () => {
+    const procs = await launchAllProcesses(config);
+    const procNames = Object.keys(procs).sort();
+    assert.deepEqual('Spawns app, chrome, and proxy',
+      ['application', 'chromedriver', 'proxy'], procNames);
+    killProcs(procs);
+  });
+
+  it('has the correct chromeOptions when NOT IN docker container', async () => {
+    const procs = await launchAllProcesses(config);
+    const { args } = config.desiredCapabilities.chromeOptions;
+    assert.deepEqual(chromeOptions, args);
+    killProcs(procs);
+  });
+
+  it('runs has --no-sandbox when run as root', async () => {
+    const getuidStub = sinon.stub(process, 'getuid').returns(0);
+
+    const procs = await launchAllProcesses(config);
+    const { args } = config.desiredCapabilities.chromeOptions;
+    assert.deepEqual([...chromeOptions, '--no-sandbox'], args);
+    getuidStub.restore();
+    killProcs(procs);
+  });
+
+  it('has the correct chromeOptions when IN docker container', async () => {
+    // simulate docker file system
+    const vol = Volume.fromJSON({
+      '/.dockerenv': 'docker',
+    });
+    ufs.use(fs).use(vol); // build fileystem
+    const unpatch = patchFs(ufs); // patch native fs calls
+
+    const procs = await launchAllProcesses(config);
+    const { args } = config.desiredCapabilities.chromeOptions;
+    assert.deepEqual([...chromeOptions, '--disable-dev-shm-usage'], args);
```

Comment:
I could also add the `--no-sandbox` flag here as well since we are running as root in docker